src/atoms/TextInput/Label.native.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Platform } from 'react-native';
+import styled from 'styled-components/native';
+import Text from '../Text';
+import View from '../View/View.native';
+import Space from '../Space';
+
+const styles = {
+  container: {
+    paddingTop({ inputLayoutDimensions }) {
+      // For aligning left label to the center of Text Field
+      if (Platform.OS === 'android') {
+        return `${inputLayoutDimensions.height * 0.36}px`;
+      }
+      return `${inputLayoutDimensions.height * 0.29}px`;
+    },
+  },
+  padding() {
+    return [0, 3, 0, 0];
+  },
+};
+
+const LabelContainer = styled(View)`
+  padding-top: ${styles.container.paddingTop};
+`;
+
+const Label = ({ children, inputLayoutDimensions }) => {
+  return (
+    <LabelContainer inputLayoutDimensions={inputLayoutDimensions}>
```

Comment:
Where are we passing this prop `inputLayoutDimensions`? React Native `View` doesn't support it.

Reply:
Passing it as a prop here so that it's available on line 16 for styling